packages/ember-htmlbars/tests/integration/streams_test.js

```diff
@@ -0,0 +1,56 @@
+import Helper from "ember-htmlbars/system/helper";
+import { registerHelper } from "ember-htmlbars/helpers";
+import run from 'ember-metal/run_loop';
+import EmberView from 'ember-views/views/view';
+import EmberObject from 'ember-runtime/system/object';
+import compile from 'ember-template-compiler/system/compile';
+import { runAppend, runDestroy } from "ember-runtime/tests/utils";
+import { set } from 'ember-metal/property_set';
+import { isStream } from 'ember-metal/streams/utils';
+
+var view;
+
+QUnit.module('ember-htmlbars: streams', {
+  setup() {
+
+  },
+
+  teardown() {
+    runDestroy(view);
+    view = null;
+  }
+});
+
+QUnit.test("Unbound helpers should be unbound when accessed directly and when accessed in a subexpression", function() {
```

Comment:
Much clearer!  Sorry for being pedantic on the naming.  I have recently been going through some older tests where their intent is extremely hard to understand from the test title, so I think I am a tad over-sensitive here.  :shaved_ice: 

Thank you!